index.html

```diff
@@ -206,26 +206,29 @@ <h2>About me</h2>
         I'm always interested in hearing about new projects, so if you'd like to
         chat please get in touch.
       </p>
-
-      <form action="https://formspree.io/f/moqzolvp" method="post">
+ 
+      <form action="https://formspree.io/f/moqzolvp" method="post" id='form'>
```

Comment:
- [ ] **[OPTIONAL]** Please consider deleting your `deskcontact` inputs. This is because it's impossible to submit the form on the desktop version because the `Fullname` input is required but isn't shown on the screen. Instead, consider having just one `Fullname` input for mobile and desktop versions! 🤓 🔧